packages/ui/src/Introduction.stories.mdx

```diff
@@ -0,0 +1,211 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
Do you want to keep this file? I'd probably deleted it.

Reply:
Only if we have something to add there. Currently we have no content for it so I will delete it and we will recover it if we will have somthing